ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```
@@ -1749,8 +1749,8 @@ periodics:
     slack:
       channel: '#announce-logging'
       job_states_to_report:
-      - failure
-      report_template: Job {{.Spec.Job}} failed.
+      - error
```

Don't you want both failure and error states?
```suggestion
      - failure
      - error
```